lib/rubocop/cop/rspec/file_path.rb

```diff
@@ -160,6 +159,14 @@ def relevant_rubocop_rspec_file?(_file)
         def spec_suffix_only?
           cop_config['SpecSuffixOnly']
         end
+
+        def routing_spec_path?
+          expanded_file_path.include?('spec/routing/')
+        end
```

Comment:
I believe this is enough in most cases, but perhaps it should be `'/spec/routing/'` or more strict check.